src/arm/bbb-bone-buses.dtsi

```diff
@@ -0,0 +1,474 @@
+		};
+
+		led_P9_11 {
+			gpios = <&gpio0 30 0>;
+			status = "disabled";
+		};
+
+		led_P9_12 {
+			gpios = <&gpio1 28 0>;
+			status = "disabled";
+		};
+
+		led_P9_13 {
+			gpios = <&gpio0 31 0>;
+			status = "disabled";
+		};
+
+		led_P9_14 {
+			gpios = <&gpio1 18 0>;
+			status = "disabled";
+		};
+
+		led_P9_15 {
+			gpios = <&gpio1 16 0>;
+			status = "disabled";
+		};
+
+		led_P9_16 {
+			gpios = <&gpio1 19 0>;
+			status = "disabled";
+		};
+
+		led_P9_17 {
+			gpios = <&gpio0 5 0>;
+			status = "disabled";
+		};
+
+		led_P9_18 {
+			gpios = <&gpio0 4 0>;
+			status = "disabled";
+		};
+
+		led_P9_19 {
+			gpios = <&gpio0 13 0>;
+			status = "disabled";
+		};
+
+		led_P9_20 {
+			gpios = <&gpio0 12 0>;
+			status = "disabled";
+		};
+
+		led_P9_21 {
+			gpios = <&gpio0 3 0>;
+			status = "disabled";
+		};
+
+		led_P9_22 {
+			gpios = <&gpio0 2 0>;
+			status = "disabled";
+		};
+
+		led_P9_23 {
+			gpios = <&gpio1 17 0>;
+			status = "disabled";
+		};
+
+		led_P9_24 {
+			gpios = <&gpio0 15 0>;
+			status = "disabled";
+		};
+
+		led_P9_25 {
+			gpios = <&gpio3 21 0>;
+			status = "disabled";
+		};
+
+		led_P9_26 {
+			gpios = <&gpio0 14 0>;
+			status = "disabled";
+		};
+
+		led_P9_27 {
+			gpios = <&gpio3 19 0>;
+			status = "disabled";
+		};
+
+		led_P9_28 {
+			gpios = <&gpio3 17 0>;
+			status = "disabled";
+		};
+
+		led_P9_29 {
+			gpios = <&gpio3 15 0>;
+			status = "disabled";
+		};
+
+		led_P9_30 {
+			gpios = <&gpio3 16 0>;
+			status = "disabled";
+		};
+
+		led_P9_31 {
+			gpios = <&gpio3 14 0>;
+			status = "disabled";
+		};
+
+		led_P9_41 {
+			gpios = <&gpio0 20 0>;
+			status = "disabled";
+		};
+
+		led_P9_91 {
+			gpios = <&gpio3 20 0>;
+			status = "disabled";
+		};
+
+		led_P9_42 {
+			gpios = <&gpio0 7 0>;
+			status = "disabled";
+		};
+
+		led_P9_92 {
+			gpios = <&gpio3 18 0>;
+			status = "disabled";
+		};
+
+		led_A15 {
+			gpios = <&gpio0 19 0>;
+			status = "disabled";
+		};
+	};
+};
+
+// UARTs
+bone_uart_1: &uart1 {
+	symlink = "bone/uart/1";
+};
+
+bone_uart_2: &uart2 {
+	symlink = "bone/uart/2";
+};
+
+bone_uart_3: &uart3 {
+	symlink = "bone/uart/3";
+};
+
+bone_uart_4: &uart4 {
+	symlink = "bone/uart/4";
+};
+
+bone_uart_5: &uart5 {
+	symlink = "bone/uart/5";
+};
+
+// I2Cs 
+bone_i2c_1: &i2c1 {
+	symlink = "bone/i2c/1";
+};
+
+bone_i2c_2: &i2c2 {
+	symlink = "bone/i2c/2";
+};
+
+// use only when bone_i2c_2 is not in use
+bone_i2c_2a: &i2c2 {
+	// symlink = "bone/i2c/2a";
+};
+
+// use only when bone_i2c_1 is not in use
+bone_i2c_3: &i2c1 {
+	// symlink = "bone/i2c/3";
+};
+
+// SPIs
+bone_spi_0: &spi0 {
+	#address-cells = <1>;
+	#size-cells = <0>;
+
+	channel@0 {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		compatible = "spidev";
+		symlink = "/bone/spi/0.0";
+
+		reg = <0>;
+		spi-max-frequency = <16000000>;
+		spi-cpha;
+	};
+};
+
+bone_spi_1: &spi1 {
+	#address-cells = <1>;
+	#size-cells = <0>;
+
+	ti,pio-mode; /* disable dma when used as an overlay, dma gets stuck at 160 bits... */
```

Comment:
I find it rather doubtful that this issue exists on both devices. Where did you copy and paste this from? Does it belong here or, more likely, in the device base-tree definition? If it specific to overlays, as is indicated, more information really is needed here.

Reply:
I am sorry, this was just meant for testing purpose. I will fix this. 
The code will be shifted to overlays just like we had for old overlays.